client/my-sites/hosting/php-version-card/style.scss

```diff
@@ -0,0 +1,11 @@
+.php-version-card__set-version {
+	margin-top: 16px;
+}
+
+.php-version-card__version-select {
+	width: 50%;
```

Comment:
Do we need the select to take 50% of the width? The default size (no `width` defined at all) seems to be enough.

Reply:
It looked like that on the design from @olaolusoga. Not required, fwiw.

Reply:
Tho it looks better at 50%. :)